engine/src/main/scala/cromwell/engine/backend/io/filesystem/gcs/GcsFileSystemProvider.scala

```diff
@@ -183,13 +183,12 @@ class GcsFileSystemProvider private (storageClient: Try[Storage], executionConte
         }
 
         withRetry(moveInner)
-      case _ => throw new UnsupportedOperationException(s"Can only copy from GCS to GCS: $source or $target is not a GCS path")
+      case _ => throw new UnsupportedOperationException(s"Can only move from GCS to GCS: $source or $target is not a GCS path")
```

Comment:
We rename stdout/stderr/call logs files when copying them in call caching so they have the expected name in the new call context. Rename is considered a `move`